css/index.css

```diff
@@ -136,11 +205,154 @@ footer nav {
     display: flex;
     justify-content: space-between;
     align-items: center;
-    padding: 20px 2%;
-    font-size: 14px;
+    padding: 2rem 2%;
+    font-size: 1.4rem;
 }
 
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/* TABLET 768PX */
+@media (max-width: 768px) {
```

Comment:
I have a horizontal scrolling bar for this viewing mode, see if you can fix this 😄